From 4ae1577b432c1859704d2763f70a764dee23bd98 Mon Sep 17 00:00:00 2001
From: Kylie McClain <somasis@exherbo.org>
Date: Thu, 28 Apr 2016 22:40:32 -0400
Subject: [PATCH] uxcons: Fix on musl libc

musl libc exports FD_SET and friends in sys/select.h.

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
Patch status: upstream commit 4ae1577b432c

 unix/uxcons.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/unix/uxcons.c b/unix/uxcons.c
index b9fd67fe9274..f2a25f3fed70 100644
--- a/unix/uxcons.c
+++ b/unix/uxcons.c
@@ -12,6 +12,7 @@
 #include <termios.h>
 #include <unistd.h>
 #include <fcntl.h>
+#include <sys/select.h>
 
 #include "putty.h"
 #include "storage.h"
-- 
2.8.1

